From fbb3e17eb16d3beba52107c53529c321bd5b725d Mon Sep 17 00:00:00 2001
From: Julie Jeongeun Kim <jkim@igalia.com>
Date: Wed, 17 Oct 2018 14:04:02 +0900
Subject: [PATCH] Remove Restart condition

It causes continue restarting. Even if it's removed,
it's launched without problem  whenever users select WebApp.
---
 files/launch/WebAppMgr.service | 1 -
 1 file changed, 1 deletion(-)

diff --git a/files/launch/WebAppMgr.service b/files/launch/WebAppMgr.service
index 5a5fe0a..d4461d5 100644
--- a/files/launch/WebAppMgr.service
+++ b/files/launch/WebAppMgr.service
@@ -20,7 +20,6 @@ Type=simple
 OOMScoreAdjust=-1000
 EnvironmentFile=-/etc/default/WebAppMgr.env
 ExecStart=/usr/bin/WebAppMgr --no-sandbox --remote-debugging-port=9998 --user-data-dir="/home/0/wamdata" --webos-wam
-Restart=on-failure
 
 [Install]
 WantedBy=multi-user.target
-- 
2.14.1

